vault.go

```diff
@@ -118,7 +118,7 @@ func doMount(vc *vault.Client, name string) (string, error) {
 	return mountName, nil
 }
 
-func doKubernetesBackendRole(vc *vault.Client, authPath, name, roleName string, policies []string) error {
+func doKubernetesBackendRole(vc *vault.Client, authPath, namespace, roleName string, policies []string) error {
```

Comment:
Can we rename `namesapce` to `profileName`, since that's technically what it is.

Reply:
Absolutely!

Reply:
Absolutely!